### PR TITLE
feat: add show-projects command to commentOps

### PR DESCRIPTION
**Pull request by toVersus**
Fixes: #547 

This PR implements new command `digger show-projects` to show all the impacted projects by the PR on the comment. 

- Address library bump issue introduced by https://github.com/diggerhq/digger/pull/544
- Add interim implementation of GetBranchName method (See https://github.com/diggerhq/lib-orchestrator/commit/65be3864de9d21f05bece8ead9826169186c7a6e)

### CODE REVIEW

File: cmd/digger/main.go [hunk 1]
```diff
@@ -238,7 +239,8 @@ func gitHubCI(lock core_locking.Lock, policyChecker core_policy.Checker, backend
 		if err != nil {
 			reportErrorAndExit(githubActor, fmt.Sprintf("Failed to process GitHub event. %s", err), 6)
 		}
-		logImpactedProjects(impactedProjects, prNumber)
```
Comment (motatoes) [inline cmd/digger/main.go:241]: Lets use a more descriptive function name. We can call it `getImpacagedProjectsAsString`

**Review by motatoes — CHANGES_REQUESTED** [review]
Looks good to me, we just need to include the function name and it will be good to go 👍

**Review by motatoes — CHANGES_REQUESTED** [review]
Please update the version of the orchestrator in your change by running ` go get github.com/diggerhq/lib-orchestrator@b6adc1f775cad261ce326c077ea14117f222df2e` so we use the correct version

File: cmd/digger/main.go (hunk 1 above)
Comment (toVersus) [reply cmd/digger/main.go:241]: Done

**Comment by toVersus** [comment]
Thank you for taking the time to look at this! I added the following commits to address the additional issue:

- Address library bump issue introduced by https://github.com/diggerhq/digger/pull/544 (https://github.com/diggerhq/digger/pull/548/commits/b97a4a70f26a3be990511e3c46236033dc5d49ab)
- Add interim implementation of GetBranchName method introduced by https://github.com/diggerhq/lib-orchestrator/commit/65be3864de9d21f05bece8ead9826169186c7a6e (https://github.com/diggerhq/digger/pull/548/commits/8d73e5e1e71131d9ecc27a402bda350d27fadc7b)

**Comment by motatoes** [comment]
@toVersus a few more tests have failed ... could you please take a look

**Comment by toVersus** [comment]
@motatoes 
OK, I figured out the route cause of the test failure. It looks like the version mismatch of google/go-github between digger and lib-orchestrator. Please see the https://github.com/diggerhq/lib-orchestrator/pull/3 🙏

**Comment by motatoes** [comment]
@toVersus thanks for the investigation! I just merged it, lets try to bump the version now and rerun the tests

(in the future we will figure out a better way to manage this dependencies and versions)

**Comment by toVersus** [comment]
@motatoes 
Thanks! I just bumped the lib-orchestrator version.

**Review by motatoes — APPROVED** [review]

**Comment by motatoes** [comment]
Thank you so much for fixing the tests and helping with the version bump in your PR. We are going to merge this and its expected to go live in the next release sometime next week!

**Pull request merged**